js/PDBLoaderImproved.js

```diff
@@ -358,19 +371,18 @@ catch(e){
 				{
 					// don't bond to other chains
 					var neighbor = molecule.atoms[j];
-					if(neighbor.chainId !== atom.chainId
-						|| Math.abs(neighbor.seqRes - atom.seqRes) > 1
-					)
-						continue;
-
-					var v2 = new THREE.Vector3(neighbor.x, neighbor.y, neighbor.z);
```

Comment:
We don't actually need a vector at this point. distanceToSquared is happy to accept any object with x, y, z members.